### PR TITLE
Provide default lang — 'ru'.

### PR DESCRIPTION
**Pull request by Yeti-or**

### CODE REVIEW

File: generators/i18n/index.js [hunk 1]
```diff
@@ -1,20 +1,24 @@
...
-        }, ['(function() {', `var core = require('${requiredPath(path.join(__dirname, "core"))}');`])
+        }, [
+            '(function() {',
+            `var core = require('${requiredPath(path.join(__dirname, "core"))}');`
+        ])
         .concat([
-            'console.error("Define process.env.BEM_LANG");',
+            'process.env.BEM_LANG?',
+            'console.error(\'bemLoader option langs should include\', "\'" + process.env.BEM_LANG + "\'"):',
+            'console.error(\'Define process.env.BEM_LANG\');',
```
Comment (collapsus) [inline generators/i18n/index.js]: message should  be the same for both cases

File: index.js [hunk 2]
```diff
@@ -24,6 +24,7 @@ module.exports = function(source) {
         ),
         levels = options.levels,
         techs = options.techs || ['js'],
+        langs = options.langs || ['ru'],
```
Comment (collapsus) [inline index.js]: 'en' by default

File: generators/i18n/index.js [hunk 3]
```diff
@@ -1,20 +1,24 @@
...
             return core()
                 .decl(require('${requiredPath(path.join(file.path, lang))}'))
-                ('${file.cell.entity.id}');
+                    ('${file.cell.entity.id}');
         }`;
-
     return files
         .reduce((acc, file) => {
             return acc.concat(langs.map(lang => strLang(file, lang)));
-        }, ['(function() {', `var core = require('${requiredPath(path.join(__dirname, "core"))}');`])
```
Comment (collapsus) [inline generators/i18n/index.js]: unnecessary

File: index.js (hunk 2 above)
Comment (zxqfox) [inline index.js]: +1

File: generators/i18n/index.js [hunk 4]
```diff
@@ -1,20 +1,24 @@
...
+        if (process.env.BEM_LANG ? process.env.BEM_LANG === '${lang}' : 'ru' === '${lang}') {
             return core()
                 .decl(require('${requiredPath(path.join(file.path, lang))}'))
-                ('${file.cell.entity.id}');
+                    ('${file.cell.entity.id}');
         }`;
-
     return files
         .reduce((acc, file) => {
             return acc.concat(langs.map(lang => strLang(file, lang)));
```
Comment (zxqfox) [inline generators/i18n/index.js]: Using reduce with `return res.concat` is a kind of faping. ;-(

File: index.js (hunk 2 above)
Comment (awinogradov) [inline index.js]: +1

File: generators/i18n/index.js (hunk 4 above)
Comment (Yeti-or) [inline generators/i18n/index.js]: faping ?

Comment (zxqfox) [inline generators/i18n/index.js]: fap-fap

Comment (Yeti-or) [inline generators/i18n/index.js]: you prefer `map().concat` ?

**Pull request merged**